[2025-06-29T14:48:25.901+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-13T00:00:00+00:00 [queued]>
[2025-06-29T14:48:26.025+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-13T00:00:00+00:00 [queued]>
[2025-06-29T14:48:26.033+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-06-29T14:48:26.485+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-07-13 00:00:00+00:00
[2025-06-29T14:48:26.529+0000] {standard_task_runner.py:57} INFO - Started process 2093 to run task
[2025-06-29T14:48:26.549+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2024-07-13T00:00:00+00:00', '--job-id', '3292', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp3jz77i_j']
[2025-06-29T14:48:26.626+0000] {standard_task_runner.py:85} INFO - Job 3292: Subtask insert_into_table
[2025-06-29T14:48:26.837+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-13T00:00:00+00:00 [running]> on host f561bfe7dd3a
[2025-06-29T14:48:27.306+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-13T00:00:00+00:00'
[2025-06-29T14:48:27.307+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-07-13', 'dag_with_postgres_operator_v03')
        
[2025-06-29T14:48:27.349+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T14:48:27.388+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T14:48:27.402+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-07-13', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-06-29T14:48:27.406+0000] {sql.py:427} INFO - Rows affected: 1
[2025-06-29T14:48:27.433+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20240713T000000, start_date=20250629T144825, end_date=20250629T144827
[2025-06-29T14:48:27.532+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-29T14:48:27.607+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
